app/jobs/get_usps_waiting_proofing_results_job.rb

```diff
@@ -0,0 +1,41 @@
+class GetUspsWaitingProofingResultsJob < GetUspsProofingResultsJob
+
+  queue_as :long_running
+
+  def perform(_now)
+    return true unless ipp_enabled? && ipp_ready_job_enabled?
+
+    @enrollment_outcomes = {
+      enrollments_checked: 0,
+      enrollments_errored: 0,
+      enrollments_expired: 0,
+      enrollments_failed: 0,
+      enrollments_in_progress: 0,
+      enrollments_passed: 0,
+    }
+
+    reprocess_delay_minutes = IdentityConfig.store.
+      get_usps_proofing_results_job_reprocess_delay_minutes
+    enrollments = InPersonEnrollment.needs_status_check_on_waiting_enrollments(
```

Comment:
is the only difference btwn these 2 jobs which enrollments are being checked?

Reply:
Yep. And how frequently they are run.